### PR TITLE
Tests for AO3-3468

### PR DESCRIPTION
**Pull request by cesy**
Checking the key behaviour, making it clear and easy for someone to fix later.

https://otwarchive.atlassian.net/browse/AO3-3468

### CODE REVIEW

File: features/users/user_rename.feature [hunk 1]
```diff
@@ -110,3 +110,30 @@ Feature:
...
+    When I visit the change username page for oldusername
+      And I fill in "New user name" with "newusername"
+      And I fill in "Password" with "password"
+      And I press "Change User Name"
+    Then I should get confirmation that I changed my username
+    When I am on the the works page
+    Then I should see "newusername"
+      And I should see "Epic story"
+      And I should not see "oldusername"
+    # doesn't work if search indexes are not updated
```
Comment (sarken) [inline features/users/user_rename.feature]: I'm slightly confused by this -- do you mean that once the bug is fixed, it _should_ work even if the search index hasn't updated, or is this just essentially a note about unfortunate but expected behavior?

Comment (cesy) [inline features/users/user_rename.feature]: It's a note about unfortunate but expected behaviour at this point - would a clearer phrasing be "Has old name until indexes are updated"?

Comment (sarken) [inline features/users/user_rename.feature]: That phrasing would be perfect, yes -- thanks!

**Comment by cesy** [comment]
Done.

**Pull request merged**